lib/tests/modules/declaration-positions.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, options, ... }:
+let discardPositions = lib.mapAttrs (k: v: v);
+in
+{
+  imports = [
+    {
+      options.imported.line7 = lib.mkOption {
+        type = lib.types.int;
+      };
+
+      # Simulates various patterns of generating modules such as
+      # programs.firefox.nativeMessagingHosts.ff2mpv. We don't expect to get
+      # line numbers for these, but we can fall back on knowing the file.
```

Comment:
Depends on the question in the other comment. We could fall back to the parent's position.